### PR TITLE
Prl 241 test

### PR DESCRIPTION
**Pull request by swaroopa86**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/fprl/DocumentGeneratorControllerIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.TestPropertySource;
+
+
+@SpringBootTest(classes = DocumentGeneratorControllerIntegrationTest.class)
+@AutoConfigureMockMvc
+@TestPropertySource("/application-local.properties")
```
Comment (shashisk97) [inline src/integrationTest/java/uk/gov/hmcts/reform/fprl/DocumentGeneratorControllerIntegrationTest.java]: rename property source file , to point to application.properties

File: src/integrationTest/java/uk/gov/hmcts/reform/fprl/DocumentManagementServiceImplIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+        String requestBody = ResourceLoader.loadJson(VALID_INPUT_JSON);
+        Response response = callPrlDocumentGenerator(requestBody);
+        GeneratedDocumentInfo generatedDocumentInfo = response.as(GeneratedDocumentInfo.class);
+
+        String[] dmStoreUrlParts = generatedDocumentInfo.getUrl().split("/");
+
+        String documentId = dmStoreUrlParts[dmStoreUrlParts.length - 1];
+
+        String url = "https://gateway-ccd.aat.platform.hmcts.net/documents/" + documentId; //todo url
```
Comment (shashisk97) [inline src/integrationTest/java/uk/gov/hmcts/reform/fprl/DocumentManagementServiceImplIntegrationTest.java]: move the url to property file

File: src/integrationTest/java/uk/gov/hmcts/reform/fprl/DocumentManagementServiceImplIntegrationTest.java [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+        String requestBody = ResourceLoader.loadJson(VALID_INPUT_JSON);
+        Response response = callPrlDocumentGenerator(requestBody);
+        GeneratedDocumentInfo generatedDocumentInfo = response.as(GeneratedDocumentInfo.class);
+
+        String[] dmStoreUrlParts = generatedDocumentInfo.getUrl().split("/");
+
+        String documentId = dmStoreUrlParts[dmStoreUrlParts.length - 1];
+
+        String url = "https://gateway-ccd.aat.platform.hmcts.net/documents/" + documentId; //todo url in properties
```
Comment (shashisk97) [inline src/integrationTest/java/uk/gov/hmcts/reform/fprl/DocumentManagementServiceImplIntegrationTest.java]: move url to property file and remove todo comment

File: src/integrationTest/java/uk/gov/hmcts/reform/fprl/IdamUtils.java [hunk 4]
```diff
@@ -10,30 +11,44 @@
...
+    @Value("${ccd.document.base.url}")
+    private String ccdDocumentBaseUrl;
 
-    private String testCaseworkerJwtToken;
+    private String idamUsername;
+
+    private int responseCode;
 
     public String generateNewUserAndReturnToken() {
         String username = "simulate-delivered" + UUID.randomUUID() + "@notifications.service.gov.uk";
```
Comment (shashisk97) [inline src/integrationTest/java/uk/gov/hmcts/reform/fprl/IdamUtils.java]: domain to be changed to @Testhmcts.net

**Review by shashisk97 — CHANGES_REQUESTED** [review]

File: src/integrationTest/java/uk/gov/hmcts/reform/fprl/DocumentGeneratorControllerIntegrationTest.java (hunk 1 above)
Comment (swaroopa86) [reply src/integrationTest/java/uk/gov/hmcts/reform/fprl/DocumentGeneratorControllerIntegrationTest.java]: Property file reference changed

File: src/integrationTest/java/uk/gov/hmcts/reform/fprl/DocumentManagementServiceImplIntegrationTest.java (hunk 3 above)
Comment (swaroopa86) [reply src/integrationTest/java/uk/gov/hmcts/reform/fprl/DocumentManagementServiceImplIntegrationTest.java]: Url moved to Properties

File: src/integrationTest/java/uk/gov/hmcts/reform/fprl/DocumentManagementServiceImplIntegrationTest.java (hunk 2 above)
Comment (swaroopa86) [reply src/integrationTest/java/uk/gov/hmcts/reform/fprl/DocumentManagementServiceImplIntegrationTest.java]: url moved to properties

**Pull request closed without merging**